scripts/integration-tests/e2e-prettier.sh

```diff
@@ -45,6 +45,16 @@ if [ "$BABEL_8_BREAKING" = true ] ; then
   rm tests/format/typescript/comments/mapped_types.ts
   # https://github.com/babel/babel/pull/16817
   sed -i 's/node.type !== "MemberExpression" ||/node.type !== "MemberExpression" || \/\/@ts-expect-error/g' src/language-js/utils/is-node-matches.js
+  # https://github.com/babel/babel/pull/16979
+  rm tests/format/typescript/trailing-comma/format.test.js
+  rm tests/format/typescript/enum/format.test.js
+  rm tests/format/typescript/conformance/types/enumDeclaration/format.test.js
+  rm tests/format/typescript/conformance/types/constKeyword/format.test.js
+  rm tests/format/typescript/conformance/types/ambient/format.test.js
+  rm tests/format/typescript/conformance/internalModules/importDeclarations/format.test.js
+  rm tests/format/typescript/keywords/format.test.js
+  rm tests/format/typescript/declare/format.test.js
+  rm tests/format/typescript/const/format.test.js
```

Comment:
/cc @fisker These tests will be impacted by this AST change.

Reply:
Thanks!